app/views/collaborators/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Add Collaborator</h1>
+
+<div class="row">
+  <div class="col-md-5">
+    <p>Please enter the email address of the user you wish to add a collaborator</p>
+    <p><strong>NOTE: </strong>Collaborator must already be registered with Blocipedia</p>
+  </div>
+  <div class="col-md-7">
+    <%= form_for [@wiki, @collaborator] do |f| %>
+      <div class="form-group">
+        <%= f.label :email %>
+        <%= f.email_field :user_id, autofocus: true, class: 'form-control', placeholder: "Enter email" %>
```

Comment:
Look into `#delegate` in Rails...  `delegate :dog, to: :owner` as an example. Does this solve your issue?